tests/spec/navigation-menu_spec.js

```diff
@@ -185,7 +185,9 @@ describe('Popup Nav', function() {
         expect(firstNavItem.focus).toHaveBeenCalledTimes(1);
       });
 
-      it('should navigate into submenu', function() {
+      // eslint-disable-next-line no-undef
+      xit('should navigate into submenu', function() {
+        // TODO: We may have removed "down arrow to open" feature, go ahead and remove this test if so
```

Comment:
yeah it seems we removed this functionality from nav menus, but we can probably leave this test pending until more direction on UI shell navigation is provided